tests/phpunit/integration/Core/Authentication/Clients/OAuth_ClientTest.php

```diff
@@ -375,6 +376,9 @@ public function test_get_proxy_permissions_url() {
 		$url = $client->get_proxy_permissions_url();
 		$this->assertContains( 'token=test-access-token', $url );
 		$this->assertContains( 'site_id=' . self::SITE_ID, $url );
+
+		// The URL should include the application name.
+		$this->assertContains( 'application_name=', $url );
```

Comment:
See above, this should also be asserted for the in the assertion block above where there is no site ID.

Reply:
👍

Reply:
Also in https://github.com/google/site-kit-wp/pull/1572/commits/5d03cb73b0433c8244dd2dbfaba9e4dc1281ad3a